### DISCUSSION TITLE
Bosch BTH-RM / Room thermostat II (Battery model)

### DISCUSSION THREAD

**Discussion by myrrdin81** [Koenkk/zigbee2mqtt#22585] [Devices]
Since 2day my Bosch Room Thermostat has at least in Z2mqtt an option for cooling_setpoint. This Device supports only heating and has 3 options. 
option 1: Off
option 2: Auto (runs the heating after some unknown preconfigured schedule) 
option 3: Heating

there is no cooling option available. 
and since this "update" has occurred all my automations are going haywire because the device doesn't respond how it is supposed 2 do.
<img width="660" alt="Screenshot 2024-05-10 at 22 48 54" src="https://github.com/Koenkk/zigbee2mqtt/assets/100981168/e2be0654-5996-4dbb-afbf-c703670ed7c1">
<img width="1250" alt="Screenshot 2024-05-10 at 22 50 03" src="https://github.com/Koenkk/zigbee2mqtt/assets/100981168/a77fab1d-b7aa-45cc-851b-b0754d2874fc">

After going back 2 the last z2mqtt docker image everything is back 2 normal...
<img width="732" alt="Screenshot 2024-05-10 at 23 20 47" src="https://github.com/Koenkk/zigbee2mqtt/assets/100981168/e428de79-6fe5-4f1c-accb-e31d1aecb284">
<img width="1115" alt="Screenshot 2024-05-10 at 23 21 30" src="https://github.com/Koenkk/zigbee2mqtt/assets/100981168/cf9e4c03-9a04-41d0-90a6-4bbffad1cb58">

**Comment by burmistrzak** [comment]
@myrrdin81 It might surprise you, but these Bosch thermostats do indeed support a cooling mode!
In the official Bosch app, there's already an option to switch to cooling (under "Climate Manager"), and now Z2M supports it as well. ✌️

You'll _probably_ have to tweak your automations a bit, because the previous Bosch converter was kinda rough and did not follow ZCL spec.

In case you want to use the build-in schedule with e.g. HA, there's the new Bosch-specific `operating_mode` [ schedule | manual | pause ] you can map how you like.
